src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy;
```

Comment:
```suggestion
package mate.academy.util;
```

replace to util package